servicetalk-http-api/src/main/java/io/servicetalk/http/api/HeaderUtils.java

```diff
@@ -60,6 +60,17 @@
     static final int HASH_CODE_SEED = 0xc2b2ae35;
     public static final BiFunction<? super CharSequence, ? super CharSequence, CharSequence> DEFAULT_HEADER_FILTER =
             (k, v) -> "<filtered>";
+    static final BiFunction<? super CharSequence, ? super CharSequence, CharSequence>
+            DEFAULT_DEBUG_HEADER_FILTER = (key, value) -> {
+                switch (key.toString()) {
+                    case "content-type":
+                    case "content-length":
+                    case "transfer-encoding":
```

Comment:
The header names are case-insensitive, see [FRC7230, section 3.2](https://tools.ietf.org/html/rfc7230#section-3.2).
We have a special utility `CharSequences.contentEqualsIgnoreCase(...)` for comparing header names.

Reply:
Please, also use our constants from `HttpHeaderNames` class.